RecordComparator.java

```diff
@@ -0,0 +1,12 @@
+public class RecordComparator implements Comparator<CustomerRecord>
+{
+	public int compare(CustomerRecord cr1, CustomerRecord cr2)
+	{
+		return cr1.getPriority()-cr2.getPriority();
+	}
+	
+	public boolean equals(CustomerRecord cr1)
```

Comment:
public boolean equals(PriorityComparator pc)
{
    if (this.equals(pc))
         return true;
    return false;
}